src/source/sctp/sctp_session.c

```diff
@@ -283,19 +277,12 @@ STATUS sctp_session_free(PSctpSession* ppSctpSession)
     CHK(pSctpSession != NULL, retStatus);
 
     usrsctp_deregister_address(pSctpSession);
-    /* handle issue mentioned here: https://github.com/sctplab/usrsctp/issues/147
-     * the change in shutdownStatus will trigger onSctpOutboundPacket to return -1 */
-    ATOMIC_STORE(&pSctpSession->shutdownStatus, SCTP_SESSION_SHUTDOWN_INITIATED);
```

Comment:
It's not safe to do so. sctp session need to wait a `ABORT ` packet before fully destroyed and `onSctpOutboundPacket ` might be called by libusrsctp

Reply:
Hi,
Can `onSctpOutboundPacket` be triggered by `usrsctp_close`? I've tried many times, but the `onSctpOutboundPacket` never be called after `usrsctp_close`